paddle/fluid/eager/autograd_meta.cc

```diff
@@ -13,144 +13,4 @@
-
-std::shared_ptr<GradNodeBase> EagerUtils::grad_node(
-    const egr::EagerTensor& target) {
-  return unsafe_autograd_meta(target)->GetMutableGradNode();
-}
-
-bool EagerUtils::IsLeafTensor(const egr::EagerTensor& target) {
-  std::shared_ptr<GradNodeBase> grad_node = EagerUtils::grad_node(target);
-  if (std::dynamic_pointer_cast<GradNodeAccumulation>(grad_node)) {
-    return true;
-  }
-
-  return false;
-}
-
-void EagerUtils::PassStopGradient(AutogradMeta** outs, size_t outs_num,
-                                  bool generate_grad) {
-  for (size_t i = 0; i < outs_num; ++i) {
-    if (!outs[i]) {
-      // TODO(jiabin): Add Tensor name here when we supported.
-      VLOG(0) << "Tensor is NULL";
-      continue;
-    }
-    outs[i]->SetStopGradient(generate_grad);
-  }
-}
-
-bool EagerUtils::ComputeRequireGrad(AutogradMeta** ins, size_t ins_num,
-                                    AutogradMeta** outs, size_t outs_num,
-                                    bool trace_backward) {
-  if (!trace_backward) return false;
-
-  for (size_t i = 0; i < ins_num; ++i) {
-    auto ins_stop_gradient = ins[i]->StopGradient();
-    if (!ins_stop_gradient) {
-      EagerUtils::PassStopGradient(outs, outs_num, ins_stop_gradient);
-      return true;
-    }
-  }
-  return false;
-}
-
-void EagerUtils::SetHistory(std::vector<AutogradMeta*>* autograd_metas,
-                            const std::shared_ptr<GradNodeBase>& grad_node) {
-  for (const auto& autograd_meta : *autograd_metas) {
-    autograd_meta->SetGradNode(grad_node);
-  }
-}
-
-void EagerUtils::SetHistory(AutogradMeta* autograd_meta,
-                            const std::shared_ptr<GradNodeBase>& grad_node) {
-  autograd_meta->SetGradNode(grad_node);
-}
-
-egr::EagerTensor EagerUtils::CreateTensorWithValue(
-    const pten::DDim& ddim, const paddle::platform::Place& place,
-    const pten::DataType& dtype, const pten::DataLayout& layout, double value,
-    bool is_leaf) {
-  egr::EagerTensor out = egr::EagerTensor();
-  // Init tensor's autograo_meta
-  auto meta = autograd_meta(&out);
-  FillConstAPI(value, ddim, place, dtype, layout, &out);
-
-  if (is_leaf) {
-    auto accumulation_node = std::make_shared<GradNodeAccumulation>();
-    meta->SetGradNode(accumulation_node);
-    meta->SetStopGradient(false);
-  }
-
-  return out;
-}
-
-void EagerUtils::SetMultiOutRankWithSlot(std::vector<AutogradMeta*>* targets,
-                                         size_t slot_id) {
-  // Set OutRankInfo from 0 to size of targets
-  for (size_t i = 0; i < targets->size(); i++) {
-    (*targets)[i]->SetSingleOutRankWithSlot(slot_id, i);
-  }
-}
-void EagerUtils::SetOutRankWithSlot(std::vector<AutogradMeta*>* targets,
-                                    size_t slot_id) {
-  SetMultiOutRankWithSlot(targets, slot_id);
-}
-void EagerUtils::SetOutRankWithSlot(AutogradMeta* target, size_t slot_id) {
-  target->SetSingleOutRankWithSlot(slot_id, 0);
-}
-
-}  // namespace egr
+namespace egr {}  // namespace egr
```

Comment:
remove this file since we don't need it

Reply:
Better keep it so we still have "autograd_meta" compilation target, otherwise there will be a solo "autograd_meta.h" and we can't compile it anymore.